docs/book/src/component-config-tutorial/testdata/project/Makefile

```diff
@@ -56,12 +56,8 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 fmt: ## Run go fmt against code.
 	go fmt ./...
 
-.PHONY: vet
-vet: ## Run go vet against code.
-	go vet ./...
```

Comment:
The lint will validate the vet as well.
But I am not sure if we want to remove this one.
Users can still only call vet if they wish using go 

So, I would say that I think is better we keep it as it is